gradle/japicmp.gradle

```
@@ -3,7 +3,7 @@ configurations {
 }
 
 dependencies {
-    baseline "org.testcontainers:${project.name}:1.16.0", {
+    baseline "org.testcontainers:${project.name}:1.17.0", {
```

Let's use "latest released":
```suggestion
    baseline "org.testcontainers:${project.name}:1.17.2", {
```

We should look into automating this with Dependabot and auto-merge feature (for this particular dependency)